salt/pillar/api-gateway.sls

```diff
@@ -11,17 +11,22 @@ api_gateway:
         password: kong
 
     # known API endpoints
-    # https://getkong.org/docs/0.8.x/admin-api/#add-api
+    # https://getkong.org/docs/0.10.x/admin-api/#add-api
     endpoints:
         echo:
             # taken from the demo
             upstream_url: https://httpbin.org/headers
-            request_path: /echo
-            strip_request_path: True
+            # is just 'uris' in 0.10.x
+            #request_path: /echo
```

Comment:
https://github.com/elifesciences/builder-configuration/search?q=request_path&unscoped_q=request_path

Reply:
I think it's worth to go through https://github.com/elifesciences/api-gateway-formula/blob/master/salt/_states/kong.py to see if anything changed there. I see some `params` that so will be passed transparently to the API, but there may be renamed concepts like these.

Reply:
yes, I took a look at kong.py to see if I could rename parameters before sending them off. It would help get through testing and then pillar data could be updated later